### PR TITLE
chore: add thirdparty endpoints to database seeds

### PR DESCRIPTION
**Pull request by kleyow**

### CODE REVIEW

File: migrations/20200831143314_endpointNameLength.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+/*****
```
Comment (kleyow) [inline migrations/20200831143314_endpointNameLength.js:1]: unsure what file naming scheme the migration uses :thinking: `migrate:create` give me `20200831143314...js where as other migrations are some different numbering scheme.

File: migrations/20200831143314_endpointNameLength.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+ --------------
+ ******/
+
+'use strict'
+
+exports.up = async (knex) => {
+  return await knex.schema.hasTable('endpointType').then(function(exists) {
+    if (exists) {
+      return knex.schema.alterTable('endpointType', function(t) {
+        t.string('name', 100).alter();
```
Comment (kleyow) [inline migrations/20200831143314_endpointNameLength.js:32]: Is it fine for us to make this `VARCHAR` bigger or do we need to make our endpoint names smaller > - >

Comment (eoln) [reply migrations/20200831143314_endpointNameLength.js:32]: maybe let keep it as is and shorten THIRDPARTY prefix to TP... ?

Comment (eoln) [reply migrations/20200831143314_endpointNameLength.js:32]: and CALLBACK to CB

Comment (kleyow) [reply migrations/20200831143314_endpointNameLength.js:32]: @lewisdaly @sridharvoruganti  any opinion on this? length migration vs endpoint refactor?

Comment (sridharvoruganti) [reply migrations/20200831143314_endpointNameLength.js:32]: either is fine for me, but if we rename these endpoints then code changes required in central-services-shared and thirdparty-api-adapter.

Comment (lewisdaly) [reply migrations/20200831143314_endpointNameLength.js:32]: Yeah let's shorten the names here - migrations in central-ledger should be avoided if at all possible

Comment (kleyow) [reply migrations/20200831143314_endpointNameLength.js]: done. shortened the endpoint names Thirdparty -> TP. Callback -> CB. Authorizations -> Auth.

I'll make the changes in central-services-shared and thirdparty-api-adapter after this is merged.

**Review by eoln — APPROVED** [review]
lgtm

**Pull request merged**